lib/common/ResourcePath.cpp

```diff
@@ -119,12 +121,16 @@ std::string Wasp::GetPythonDir()
 #ifdef WIN32
     return GetPythonPath();
 #endif
-
-    string path = GetResourcePath("");
-
-    string exists = FileUtils::JoinPaths({path, PYTHON_INSTALLED_PATH});
-
-    if (!FileUtils::Exists(FileUtils::JoinPaths({path, PYTHON_INSTALLED_PATH}))) path = string(PYTHON_DIR);
+    string path = string(PYTHON_DIR);  // Try our third-party-library directory first
```

Comment:
We want to use the installed python dir first, and only have the developer libs as a fallback.

Reply:
This is a fairly obscure part of our codebase.  I'd recommend revisiting it after 3.9.3 but we can postpone the release if it's that important.

Reply:
I'm not sure what you mean by obscure, this determines the version of python that is used by Vapor so it is rather critical. With this change it will be impossible for a developer to test an installed version of Vapor without first removing their third party library installation.

Reply:
I understand and totally see you point.  IMO, I don't think it's critical to fix now, but it's critical to fix soon because installers are not often tested on dev computers.  This is why we do platform tests on clean (non-dev) systems.

If this solution didn't work as in this PR, I would agree to modify it as you've suggested.

This file is obscure because:
1) It heavily relies on macro'd functions that cannot be plugged into a debugger like lldb.  GetResourcePath() can't be debugged without adding print statements to see what happens
2) PYTHON_INSTALLED_PATH is also macro'd
3) CMake code injections are furthermore specified from lib/common/CMakeConfig.cpp.in.  This is as hard to debug as macro's, but the changes are coming from yet another compile time modification
4) The #ifdef WIN32 macro for GetPythonPath() should be in-lined with GetPythonDir().  What's the difference between a PythonPath and a PythonDir?Apparently it's windows is dir, unix is path?  Path and Dir are synonyms so this makes no sense.

Reply:
I use the installers to test on my dev machine frequently when I am unsure if an issue is related to new code or my dev environment. This invalidates such a test. It also creates the situation where if someone has a different incompatible version of the third part libs installed (older or newer) vapor may try to load that version and crash. A few months down the line we may forget about this behavior and have to figure out why vapor is crashing. Since this is only going to affect developers not end users I'm fine with not holding up the release over this. Also, read obscure as *not important or well known*.